service_discovery/dto/location_test.go

```diff
@@ -85,13 +84,6 @@ func TestLocationUnserialize(t *testing.T) {
 			Location{},
 			nil,
 		},
-		{
```

Comment:
Why to reduce tests?

Reply:
This test is go version dependent.. does no work with newer go versions.

Reply:
It can be relaxed i.e. check that error starts with json: and it should be enough.

Reply:
ok, moved to separate test